SmartDeviceLink/SDLChangeRegistration.h

```diff
@@ -3,10 +3,8 @@
 
 
 #import "SDLRPCRequest.h"
-
 #import "SDLLanguage.h"
-
-@class SDLTTSChunk;
+#import "SDLTTSChunk.h"
```

Comment:
This should not be imported here, it was correct previously. If necessary in the `.m` it should be imported in the `.m`

Reply:
Fixed